src/components/YouTube.js

```diff
@@ -32,7 +32,7 @@ class YouTubeToggle extends React.Component {
 
   render() {
     if (!this.props.youTubeId) {
-      return;
+      return <div></div>;
```

Comment:
Sorry, this was a bug I introduced. I was gonna fix it, but you beat me to it. 
The correct fix is `return null;`.
I want these changes now, so I'm just gonna merge this and fix it myself.